### ISSUE TITLE
Accessibility: Format 3rd Party Gradebook progress indicator

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#649]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: Format 3rd Party Gradebook

Issue notes:
Screen reader did not announce the progress indicator or indicate that is a part of "Select - Upload - Review - Confirmation"

<img width="1048" height="133" alt="Image" src="https://github.com/user-attachments/assets/578699ba-fe27-4207-b140-9c64889c6900" />

Criterion:
